readthedocs/search/lib.py

```diff
@@ -145,7 +147,13 @@ def search_file(request, query, project_slug=None, version_slug=LATEST, taxonomy
         body['facets']['version']['facet_filter'] = final_filter
         body['facets']['taxonomy']['facet_filter'] = final_filter
 
+    if settings.DEBUG:
+        print "Before Signal"
+        pprint(body)
```

Comment:
If this is for debug on local dev, we shouldn't be using print statements, logging to debug would be best.